apps/core/src/pages/privacy-policy/components/DataAuthority.tsx

```diff
@@ -0,0 +1,31 @@
+import { Text, Title } from "@mantine/core";
+import { RefObject } from "react";
+import useStyles from "./general";
+
+type propType = {
+  innerRef: RefObject<HTMLDivElement>;
+};
+
+const DataAuthority = ({ innerRef }: propType) => {
+  const { classes } = useStyles();
```

Comment:
if you're going to reuse the styles hook multiple times(good idea btw) then you should name it better.

maybe `usePrivacyPolicyStyles`